server/data/educationAndWorkPlanClient.test.ts

```diff
@@ -23,14 +24,45 @@ describe('educationAndWorkPlanClient', () => {
       // Given
       const prisonNumber = 'A1234BC'
       const systemToken = 'a-system-token'
-      const createGoalRequest = aValidCreateGoalRequestWithOneStep()
-      educationAndWorkPlanApi.post(`/action-plans/${prisonNumber}/goals`).reply(200, createGoalRequest)
+
```

Comment:
This test was slightly wrong, in that it was using the `createGoalRequest` as the response object for the mock API .... so I've fixed it 👍